src/encoded/static/components/filegallery.js

```diff
@@ -610,7 +627,11 @@ class RawSequencingTable extends React.Component {
 
                                 return (
                                     <tr key={file['@id']} className={rowClasses.join(' ')}>
-                                        <td className="table-raw-biorep">{file.biological_replicates && file.biological_replicates.length ? file.biological_replicates.sort((a, b) => a - b).join(', ') : 'N/A'}</td>
+                                        {
+                                          showReplicateNumber ?
+                                          <td className="table-raw-biorep">{file.biological_replicates && file.biological_replicates.length ? file.biological_replicates.sort((a, b) => a - b).join(', ') : 'N/A'}</td> :
+                                          null
```

Comment:
Need four spaces indent here too. Also, while I don’t think every person needs to do the same ternary indenting style as other people, I think each person should generally be consistent with themselves. You have braces on their own line here, but on the same line as the first term in other places.